modules/self-managed-node-group/main.tf

```diff
@@ -395,6 +395,7 @@ resource "aws_launch_template" "this" {
   ]
 
   lifecycle {
+    ignore_changes = [tags, tag_specifications]
```

Comment:
isn't this a bit problematic if we change tags in the future are these tags not used by k8 at all?

Reply:
nvm docs say it does not matter